schemas/common/survey_metadata.json

```diff
@@ -78,9 +81,20 @@
         },
         "FIRST_NAME": {
           "$ref": "definitions.json#/first_name"
+        },
+        "WINDOW_START_DATE": {
+          "$ref": "definitions.json#/iso_8601_date",
+          "description": "The ISO 8601 date of COLLEX_OPEN_DATE"
+        },
+        "WINDOW_CLOSE_DATE": {
+          "$ref": "definitions.json#/iso_8601_date",
+          "description": "The ISO 8601 date of COLLEX_CLOSE_DATE"
+        },
+        "PORTAL_ID": {
+          "$ref": "definitions.json#/portal_id"
         }
       },
-      "required": ["qid"]
+      "required": ["qid", "PARTICIPANT_WINDOW_ID", "PORTAL_ID"]
```

Comment:
Q: is there any constraint on string case?

Reply:
There should have been, but our validator let through caps for metadata which is now too late to get changed.

Reply:
👍